selection_tools/include/HistogramFiller.hpp

```diff
@@ -29,12 +29,15 @@ class HistogramFiller {
   void fill_first_muon_from_alp_selection_hists(const Particle* particle, const Event *event);
   void fill_alp_in_preselection_hists(const Particle* particle_1, const Particle* particle_2, const Particle* mother, const Event *event, std::string sign);
   
+  void fill_2d_hists(const Particle* particle_1, const Particle* particle_2, std::string sign);
+
   void save_histograms(std::string output_path);
   std::map<std::string, HistogramSet*> histSets;
   
 private:
   CutsManager cutsManager;
   std::vector<double> ptCuts;
+  bool reduce_hists_=false;
```

Comment:
I just noticed it now, but we should stick to one naming convention (I was using camelCase for variables, while here you added underscores). Another thing is class members with underscore at the end. For now maybe try to match what's already there and in the future we should pick one "official" convention, like google or PEP or something and stick to it.